axelrod/tests/unit/test_match.py

```diff
@@ -32,6 +32,19 @@ def test_init(self, turns, game):
         self.assertEqual(match.noise, 0)
         self.assertEqual(match.game.RPST(), game.RPST())
 
+    @given(turns=integers(min_value=1, max_value=200), game=games())
+    @example(turns=5, game=axelrod.DefaultGame)
+    def test_init(self, turns, game):
```

Comment:
Can you change the name of this method to `test_init_with_non_default_match_attributes` or perhaps something that's less of a mouthful (but something that isn't `test_init`)?